cypress/features/regression/test/actionPopover.feature

```diff
@@ -42,15 +42,15 @@ Feature: Action Popover component in noIFrame
 
   @positive
   Scenario: verify that the first element Business is focused using Home key
```

Comment:
Could this scenario and others in the main menu level be put into an Examples table?

Reply:
nope, because those TCs have different steps to proceed